app/controllers/api/v1/users_controller.rb

```diff
@@ -5,6 +5,10 @@ module V1
     class UsersController < ApiController
       skip_before_action :ensure_authenticated, only: %i[create]
 
+      before_action only: %i[update destroy purge_avatar] do
```

Comment:
I think that `ensure_authorized('ManageUsers')` should concern only all o the actions of  `Admin::UsersController`,
The explanation is given down below,

Reply:
Admin seems to be using regular user endpoints.